jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -296,11 +298,34 @@ def apply_gpu_config(self, gpu_mode, gpu_count, pod):
     return pod
 
   @classmethod
-  def apply_pod_profile(self, spawner, pod, profile):
+  def generate_volume_path(self, mountPath, default_mount_path):
+    if (mountPath):
+      if (os.path.isabs(mountPath)):
+        return mountPath
+      return os.path.join(default_mount_path, mountPath)
+    return default_mount_path
```

Comment:
This will fail since the `default_mount_path` is already used by the default PVC we mount for every user.

It has to be `default_mount_path` joined with `volume_name`

Reply:
one note here - you will probably need to turn any special characters in the `volume_name` into `_` or something like that to make sure it does not fail when user names the volume something like `My @{}# volume with nejlepším jménem`